sklearn/feature_extraction/_hashing_fast.pyx

```diff
@@ -7,14 +7,12 @@ cimport cython
 from libc.stdlib cimport abs
 from libcpp.vector cimport vector
 
-cimport numpy as np
+cimport numpy as cnp
 import numpy as np
 from ..utils._typedefs cimport INT32TYPE_t, INT64TYPE_t
 from ..utils.murmurhash cimport murmurhash3_bytes_s32
 from ..utils._vector_sentinel cimport vector_to_nd_array
 
-np.import_array()
```

Comment:
To be safe, we still need this when using the NumPy's C API:

```python
cnp.import_array()
```

It is recommended by Cython in: https://github.com/cython/cython/blob/00ffb022f82cf13a055102288ca5c1b299f4ecdc/docs/examples/tutorial/numpy/convolve2.pyx#L12-L16 (Interacting with the `cnp.ndarray` object may use the `PyArray_*` API, which requires this call.)

Reply:
Thank you for pointing this out!